locale/shine/extensions/voterandom/deDE.json

```diff
@@ -0,0 +1,36 @@
+{
+	"SHUFFLE_RANDOM": "Stelle die Teams zufällig zusammen",
+	"SHUFFLE_SCORE": "Stelle die Teams basierend auf den Punkte pro Minute zusammen",
+	"SHUFFLE_KDR": "Stelle die Teams basierend auf den KDRs zusammen",
+	"SHUFFLE_HIVE": "Stelle die Teams basierend auf den Hive Skill Bewertungen zusammen",
+	"RANDOM_BASED_LOWER": "Zufällig",
+	"SCORE_BASED_LOWER": "Punkte basierend",
+	"KDR_BASED_LOWER": "KDR basierend",
+	"HIVE_BASED_LOWER": "Hive Skill basierend",
+	"RANDOM_BASED": "Zufällig",
+	"SCORE_BASED": "Punkte basierend",
+	"KDR_BASED": "KDR basierend",
+	"HIVE_BASED": "Hive Skill basierend",
+	"AUTO_SHUFFLE": "{ShuffleType:Translation:voterandom} wegen Servereinstellungen.",
+	"PREVIOUS_VOTE_SHUFFLE": "{ShuffleType:Translation:voterandom} wegen vorheriger Abstimmung.",
+	"NEXT_ROUND_SHUFFLE": "{ShuffleType:Translation:voterandom} für die nächste Runde...",
+	"TEAMS_FORCED_NEXT_ROUND": "Teams werden zu {ShuffleType:Translation:voterandom} gezwungen in der nächsten Runde.",
+	"SHUFFLE_AND_RESTART": "{ShuffleType:Translation:voterandom} und starte Runde neu...",
+	"SHUFFLING_TEAMS": "{ShuffleType:Translation:voterandom}...",
+	"TEAMS_SHUFFLED_FOR_DURATION": "{ShuffleType:Translation:voterandom}e Teams wurden aktiviert für die nächsten {Duration:Duration}.",
+	"TEAM_ENFORCING_TIMELIMIT": "{ShuffleType:Translation:voterandom}e Teams erzwingen wurde deaktiviert, da das Zeitlimit erreicht ist.",
+	"TEAM_SWITCH_DENIED": "Du kannst das Team nicht wechseln. {ShuffleType:Translation:voterandom}e Teams sind aktiviert.",
+	"PLACED_ON_RANDOM_TEAM": "Du wurdest in einem zufälliges Team eingeteilt.",
+	"PLACED_ON_HIVE_TEAM": "Du wurdest gemäß deiner Hive Skill Bewertung einem Team zugeteilt.",
+	"ERROR_NOT_ENOUGH_PLAYERS": "Nicht genug Spieler um eine Abstimmung zu starten.",
+	"ERROR_CANNOT_START": "Du kannst derzeit nicht für {ShuffleType:Translation:voterandom}e Teams abstimmen.",
+	"ERROR_ALREADY_ENABLED": "Es wurde bereits für {ShuffleType:Translation:voterandom}e Teams f�r die n�chste Runde abgestimmt.",
```

Comment:
I'm guessing it's not supposed to be `f�r die n�chste` here. An encoding issue, or just the wrong characters?